scripts/build.js

```diff
@@ -81,29 +76,30 @@ function buildPackage(p) {
   process.stdout.write(`${OK}\n`);
 }
 
-function buildFile(file, silent) {
-  buildFileFor(file, silent, 'node');
+function buildBrowserPackage(p) {
+  const srcDir = path.resolve(p, SRC_DIR);
+  const pkgJsonPath = path.resolve(p, 'package.json');
 
-  const pkgJsonPath = path.resolve(
-    PACKAGES_DIR,
-    getPackageName(file),
-    'package.json'
-  );
   const {browser} = require(pkgJsonPath);
   if (browser) {
     if (browser.indexOf(BUILD_ES5_DIR) !== 0) {
       throw new Error(
         `browser field for ${pkgJsonPath} should start with "${BUILD_ES5_DIR}"`
       );
     }
-    buildFileFor(file, silent, 'es5');
+    browserBuild(
+      p.split('/').pop(),
+      path.resolve(srcDir, 'index.js'),
+      path.resolve(p, browser)
+    ).then(() => {
```

Comment:
can you add a `catch` here?

`(node:21659) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: 'getMatchers' is not exported by packages/jest-matchers/src/jest_matchers_object.js` vs

```
{ Error: 'getMatchers' is not exported by packages/jest-matchers/src/jest_matchers_object.js
    at error (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:170:12)
    at Module.error$1 [as error] (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:8007:2)
    at Module.trace (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:8106:9)
    at ModuleScope.findDeclaration (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:7691:22)
    at Scope.findDeclaration (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:5497:39)
    at CallExpression.bind (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:6022:28)
    at /Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:5291:50
    at VariableDeclarator.eachChild (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:5308:5)
    at VariableDeclarator.bind (/Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:5291:7)
    at /Users/simbekkh/repos/jest/node_modules/rollup/dist/rollup.js:5291:50
  code: 'MISSING_EXPORT',
  url: 'https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module',
  pos: 1321,
  loc:
   { file: '/Users/simbekkh/repos/jest/packages/jest-matchers/src/index.js',
     line: 31,
     column: 29 },
  frame: '29: import { equals } from \'./jasmine_utils\';\n30: import { any, anything, arrayContaining, objectContaining, stringContaining, stringMatching } from \'./asymmetric_matchers\';\n31: import { getState, setState, getMatchers, se
tMatchers } from \'./jest_matchers_object\';\n                                 ^\n32: import extractExpectedAssertionsErrors from \'./extract_expected_assertions_errors\';' }
```

```diff
diff --git i/scripts/build.js w/scripts/build.js
index 5f73065c..88d084ce 100644
--- i/scripts/build.js
+++ w/scripts/build.js
@@ -94,7 +94,11 @@ function buildBrowserPackage(p) {
     ).then(() => {
       process.stdout.write(adjustToTerminalWidth(`${path.basename(p)}\n`));
       process.stdout.write(`${OK}\n`);
-    });
+    })
+      .catch(e => {
+        console.error(e);
+        process.exit(1);
+      });
   }
 }
```